.requirements

```diff
@@ -11,3 +11,4 @@ LUA_RESTY_EVENTS=2f6fa23eb3d0b76a3b35fd915711200e90bc6732 # 0.1.6
 LUA_RESTY_WEBSOCKET=60eafc3d7153bceb16e6327074e0afc3d94b1316 # 0.4.0
 ATC_ROUTER=72cc8fddeac024c54c9c1fa5a25c28a72d79080e # 1.1.0
 
+KONG_ADMIN_GUI_VERSION=nightly
```

Comment:
let's align with EE, the name would be the repository name of the dependency in upper case and `-` replaced to `_`
```suggestion
KONG_ADMIN=nightly
```